src/ott/tools/progot.py

```diff
@@ -187,6 +187,9 @@ def __init__(
           epsilon_scales
       ), "Epsilon scales have different length than alphas."
 
+    assert all(0.0 <= element <= 1.0 for element in alphas
```

Comment:
This is going to cause issues with jitting, please remove or use [this](https://jax.readthedocs.io/en/latest/debugging/checkify_guide.html);